pandas/io/parsers.py

```diff
@@ -2727,9 +2727,6 @@ def _next_iter_line(self, row_num):
                            'cannot be processed in Python\'s '
                            'native csv library at the moment, '
                            'so please pass in engine=\'c\' instead')
-                elif 'newline inside string' in msg:
```

Comment:
was this just not hit before?

Reply:
ahh ok

Reply:
or is that just really old?

Reply:
Really old, it seems.  I couldn't find it in their source code.